rmf_site_editor/src/site/light.rs

```diff
@@ -15,15 +15,117 @@
  *
 */
 
-use bevy::prelude::*;
-use rmf_site_format::{LightType, Pose};
+use bevy::{
+    pbr::CubemapVisibleEntities,
+    render::{
+        primitives::{CubemapFrusta, Frustum},
+        view::VisibleEntities,
+    },
+    prelude::{
+        *, SpotLight as BevySpotLight,
+        PointLight as BevyPointLight,
+        DirectionalLight as BevyDirectionalLight,
+    }
+};
+use rmf_site_format::LightKind;
+use crate::site::{SiteAssets, CurrentLevel};
+
+/// True/false for whether the physical lights of an environment should be
+/// rendered.
+#[derive(Clone, Copy)]
+pub struct PhysicalLightToggle(pub bool);
+
+impl Default for PhysicalLightToggle {
+    fn default() -> Self {
+        Self(true)
+    }
+}
 
-// TODO(MXG): Give a way to interact with lights and change their properties
 pub fn add_physical_lights(
     mut commands: Commands,
-    physical_lights: Query<(Entity, &LightType, &Pose), Added<LightType>>,
+    added: Query<(Entity, Option<&Parent>), Added<LightKind>>,
+    physical_light_toggle: Res<PhysicalLightToggle>,
+    current_level: Res<CurrentLevel>,
+) {
+    for (e, parent) in &added {
+        // This adds all the extra components provided by all three of the
+        // possible light bundles so we can easily switch between the different
+        // light types
+        commands
+            .entity(e)
+            .insert(Visibility { is_visible: physical_light_toggle.0 })
```

Comment:
Should we add a `Category::Light` component to have the inspector plugin show the right label? Otherwise it just shows `<Unknown Type>`

Reply:
Whoops, good catch! Fixed: [2a8d533](https://github.com/open-rmf/rmf_site/pull/89/commits/2a8d533eea61512b420ecd49ede9c189e6916795)